mon-pix/app/templates/components/campaign-banner.hbs

```diff
@@ -8,7 +8,11 @@
       <div class="campaign-banner__title"><h1>{{title}}</h1></div>
     {{/if}}
     <div class="campaign-banner__home-link">
-      {{#link-to "index" class="campaign-banner__home-link"}}‹ Quitter{{/link-to}}
+      {{#if returnToProfileV2}}
```

Comment:
Pourquoi ne pas directement appeler ce component avec la bonne route à mettre dans le link-to ?

Reply:
OK !